docs/resolvers.md

```diff
@@ -123,4 +123,41 @@ const data = cache.readFragment(gql`
 This way we'll get the Todo with id 1 and the relevant data we are askng for in the
 fragment.
 
+## Pagination
+
+Given you have a [relay-compatible schema](https://facebook.github.io/relay/graphql/connections.htm)
+on your backend we offer the possibility of endless data resolving.
+This means that when you fetch the next page in your data
+received in `useQuery` you'll see the previous pages as well. This is usefull for
+endless scrolling.
+
+You can achieve this by importing `relayPagination` from `@urql/exchange-graphcache/extras`.
+
+```js
+import { cacheExchange } from '@urql/exchange-graphcache';
+import { relayPagination } from '@urql/exchange-graphcache/extras';
+
+const cache = cacheExchange({
+  resolvers: {
+    Query: {
+      todos: relayPagination(),
+    },
+  },
+});
+```
+
+`relayPagination` accepts an object of options, for now we are offering one
+option and that is the `mergeMode`. This defaults to `inwards` and can otherwise
+be set to `outwards`. This will determine how your edges are merged, from right to left
```

Comment:
Right-to-left and left-to-right is ambiguous since it’s possible to paginate in reverse.

> This will handle how pages are merged when you paginate forwards and backwards at the same time. `outwards` pagination assumes that pages that come in `last` should be merged before the `first` pages, so that the list grows outwards in both directions. The default `inwards` pagination assumes that pagination `last` pages is part of the same list and come after `first pages`. Hence it merges pages so that they converge in the middle.